portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,32 +33,53 @@
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-  private static final String[] COMMENTS = {
-    "Wow good job on the first portfolio website",
-    "You should find some help, the design is wacky",
-    "Where am I?"
-  };
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String json = convertArrayToJson(COMMENTS);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    String json = getDataStoreAll(datastore);
+
     response.setContentType("text/html;");
```

Comment:
Here and bellow, the content type can be "application/json"

https://developer.mozilla.org/en-US/docs/Web/HTTP/Basics_of_HTTP/MIME_types/Common_types

Reply:
done